<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAudit</class>
 <widget class="QWidget" name="ItemAudit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
    #frame_card {
        background-color: white;
        border: 1px solid #E2E8F0;
        border-radius: 8px;
    }
    #lbl_action {
        color: #1E293B;
        font-weight: bold;
        font-size: 14px;
    }
    #lbl_details {
        color: #64748B;
        font-size: 12px;
    }
    #lbl_user {
        color: #334155;
        font-weight: 600;
        font-size: 11px;
        background-color: #F1F5F9;
        padding: 4px 8px;
        border-radius: 4px;
    }
    #lbl_time {
        color: #94A3B8;
        font-size: 11px;
    }
   </string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_card">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_icon">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>background-color: #F8FAFC; border-radius: 20px; border: 1px solid #E2E8F0;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>

      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_action">
          <property name="text">
           <string>Action Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_details">
          <property name="text">
           <string>Details of the action performed goes here...</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_user">
          <property name="text">
           <string>User Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_time">
          <property name="text">
           <string>2023-10-25 10:30 AM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
</ui>